examples/ui-demo/src/app/config.tsx

```diff
@@ -10,8 +10,12 @@ export type Config = {
     showExternalWallets: boolean;
     showPasskey: boolean;
     addPasskey: boolean;
-    showSocial: boolean;
-    addGoogleAuth: boolean;
+    showOAuth: boolean;
+    oAuthMethods: {
+      google: Boolean;
+      facebook: Boolean;
+      auth0: Boolean;
```

Comment:
Should use `boolean` over `Boolean`: there basically isn't ever any reason to use `Boolean`.

Reply:
good catch